src/diagapp.jl

```diff
@@ -1,25 +1,37 @@
 # Diagonal Approximation for trace of inverse of matrix
+# Values from Extrapolation Methods for Estimating Trace of the Matrix Inverse by P. Fika
 
-export diagapp
+export diagapp, diagapp!
 
 using LinearAlgebra
 
+struct diagappspace
```

Comment:
Convention in struct names is upper case first letter.

Reply:
Okay, in my head I actually left it like that since I was gonna keep it for internal use only.